lib/services/debug-service.ts

```diff
@@ -0,0 +1,65 @@
+import { platform } from "os";
+import { EventEmitter } from "events";
+import { CONNECTION_ERROR_EVENT_NAME } from "../constants";
+
+class DebugService extends EventEmitter {
```

Comment:
This debug service has different debug method. In ios/android debug service the debug method returns array of the chrome devtools urls. In this service we want to start debugging only on one device and return the url only for it.